conda/prod.yaml

```diff
@@ -20,7 +20,7 @@ dependencies:
   - python-dotenv
   - setuptools
   - sqlalchemy
-  - superset >=1.4.1
+  - superset 1.5.0
```

Comment:
shouldn't ths be `superset==1.5.0` or  `superset>=1.5.0

Reply:
it is not necessary to use `==`
and as we are using conda lock .. `superset>=1.5.0`will not be totally useful as I will need to rebuild this every time a new superset will be released.

also as superset is the base app here, I think that makes sense to have a hard pinning here because we need to check also other pinnings and test it on staging before having it on production